### PR TITLE
ticket 15: added functionality and testing

### PR DESCRIPTION
**Pull request by jd4545**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -220,4 +220,39 @@ describe("News app",()=>{
             })
         })
     })
+    describe('GET /api/articles/:article_id/comments', () => {
```
Comment (rogersop) [inline __tests__/app.test.js:222]: One more test cases to consider: What should we respond with for an article that has no comments?

**Review by rogersop — CHANGES_REQUESTED** [review]
Nice work, just one more test case to consider here 😄

**Comment by jd4545** [comment]
Note: Had a conversation with Paul in Slack about what the client was receiving in body.comments. This was in the format  [ [all the comment objects], null], which always has length 2. I resolved this by changing the controller, where the issue seemed to originate. I suspect from the Promise.all, but am not 100% sure. In any case I sent the 1st element of the array back to the client which was an array of comment objects.

**Review by Augs0 — APPROVED** [review]
I don't think this PR was resubmitted so Paul probably didn't see your responses and refactors

**Pull request merged**